tools/bootstrap-plugin-cache.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -euo pipefail
+
+# Disable spinner even when we have a TTY
+export CI='1'
+
+find ./examples -type f -name '*cdktf.json' |  tr '\n' " " | xargs jq -s "map(.terraformProviders) | add | unique | { \"language\": \"typescript\", \"terraformProviders\": . }" > bootstrap.json
+
+scriptdir=$(cd $(dirname $0) && pwd)
+
+cd $(mktemp -d)
+mkdir test && cd test
+
+cdktf init --template typescript-minimal --project-name="typescript-bootstrap" --project-description="typescript bootstrap plugin cache" --local
+
+# add null provider
+mv ${scriptdir}/../bootstrap.json ./cdktf.json
```

Comment:
As we expand what's in `cdktf.json`, I could see this overwriting some useful information.

Reply:
It's only about downloading the providers to warm up the cache. I don't think there will be any custom logic involved.